services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1292,10 +1295,40 @@ static void emitPushJobStatusMetrics(
    */
   @Override
   public void sendPushJobDetails(PushJobStatusRecordKey key, PushJobDetails value) {
+    sendPushJobDetailsToLocalRT(key, value);
+    if (isParent()) {
+      String lastDualWriteError = "";
+      for (Map.Entry<String, ControllerClient> entry: getControllerClientMap(getPushJobStatusStoreClusterName())
+          .entrySet()) {
+        LOGGER.info("Sending push job details: {} to region: {} for: {}", value, entry.getKey(), key);
+        ControllerResponse response = entry.getValue()
+            .sendPushJobDetails(
+                key.storeName.toString(),
+                key.versionNumber,
+                getPushJobDetailsSerializer().serialize(null, value));
+        if (response.isError()) {
+          LOGGER.error(
+              "Unable to send push job details to region: {}, for key: {}, value: {} for error: {}",
+              entry.getKey(),
+              key,
+              value,
+              response.getError());
+          lastDualWriteError = response.getError();
+        } else {
+          // When A/A is enabled, we only need to write to one child region and it will be replicated to all regions.
+          return;
+        }
+      }
+      throw new VeniceException(
+          "Unable to dual write push job details to any child region with last error:" + lastDualWriteError);
```

Comment:
This looks a little weird unless I'm counting my brackets wrong.  This looks like it'll error if this a child controller.  Shouldn't we update the message to include that case?

Reply:
Nevermind, I counted the brackets wrong